### PR TITLE
[zh-cn] resync: determine-reason-pod-failure.md

### PR DESCRIPTION
**Pull request by hs0210**
Signed-off-by: Hu Shuai <hus.fnst@fujitsu.com>

### CODE REVIEW

File: content/zh-cn/docs/tasks/debug/debug-application/determine-reason-pod-failure.md [hunk 1]
```diff
@@ -126,6 +126,26 @@ Kubernetes 从容器的 `terminationMessagePath` 字段中指定的终止消息
...
+You cannot set the termination message path after a Pod is launched
+-->
+终止消息旨在简要说明最终状态，例如断言失败消息。 
+kubelet 会截断长度超过 4096 字节的消息。
+
+所有容器的总消息长度限制为 12KiB，将会在每个容器之间平均分配。 
+例如，如果有 12 个容器（`initContainers` 或 `containers`）， 
+每个容器都有 1024 字节的可用终止消息空间。
+
+默认的终止消息路径是`/dev/termination-log`。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/debug/debug-application/determine-reason-pod-failure.md:146]:
```suggestion
默认的终止消息路径是 `/dev/termination-log`。 
```

**Review by howieyuen — APPROVED** [review]
/approve

File: content/zh-cn/docs/tasks/debug/debug-application/determine-reason-pod-failure.md (hunk 1 above)
Comment (hs0210) [reply content/zh-cn/docs/tasks/debug/debug-application/determine-reason-pod-failure.md]: Done. PATL.

**Comment by ydFu** [comment]
thanks.
/lgtm

**Pull request merged**